java/fury-core/src/main/java/org/apache/fury/config/Config.java

```diff
@@ -60,6 +60,7 @@ public class Config implements Serializable {
   private final boolean scalaOptimizationEnabled;
   private transient int configHash;
   private final boolean deserializeNonexistentEnumValueAsNull;
+  private final boolean treatEnumAsString;
```

Comment:
could we rename it as `serializeEnumByName` instead to keep more consistent naming style?